src/clientmetricreport/RawMetricReport.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default class RawMetricReport {
+  previousMetrics: RTCStatsReport;
```

Comment:
Why do we need to provide `previousMetrics`? Could avoid creating a wrapper object? easy for builder to do themselves.

Reply:
Removed this unnecessary abstraction.